src/features/user/MFV2/ContributionsMap/Markers/SinglePointMarkers.tsx

```diff
@@ -1,13 +1,13 @@
 import { Marker } from 'react-map-gl-v7';
 import { useMemo } from 'react';
-import RegisteredTreeIcon from '../../../../../../public/assets/images/icons/myForestV2Icons/RegisteredTreeIcon';
-import NaturalRegeneration from '../../../../../../public/assets/images/icons/myForestV2Icons/NaturalRegeneration';
-import Mangroves from '../../../../../../public/assets/images/icons/myForestV2Icons/Mangroves';
-import ManagedRegeneration from '../../../../../../public/assets/images/icons/myForestV2Icons/ManagedRegeneration';
-import Agroforestry from '../../../../../../public/assets/images/icons/myForestV2Icons/Agroforestry';
-import UrbanRestoration from '../../../../../../public/assets/images/icons/myForestV2Icons/UrbanRestoration';
-import TreePlanting from '../../../../../../public/assets/images/icons/myForestV2Icons/TreePlanting';
-import OtherPlanting from '../../../../../../public/assets/images/icons/myForestV2Icons/OtherPlanting';
+import RegisteredTreeIcon from '../../../../../../public/assets/images/icons/myForestV2Icons/markerIcon/RegisteredTreeIcon';
```

Comment:
For cleaner code, combine these icons in the same file which may be `ProfileV2Icons` or a new file.